lib/glimesh_web/templates/oauth2_provider/authorized_application/index.html.eex

```diff
@@ -0,0 +1,25 @@
+<div class="container" style="text-align:center;">
+  <h1>Authorized Applications</h1>
+
+  <table style="width:50%; margin: 0 25%">
+    <thead>
+      <tr class="row">
+        <th class="col-md-5 list-group-item">Application</th>
+        <th class="col-md-4 list-group-item">Created At</th>
+        <th class="col-md-3"></th>
+      </tr>
+    </thead>
+    <tbody>
+      <%= for application <- @applications do %>
+        <tr class="row">
+          <td class="col-md-5 list-group-item"><%= application.name %></td>
+          <td class="col-md-4 list-group-item"><%= application.inserted_at %></td>
+
+          <td class="col-md-3 list-group-item">
+            <span><%= link "Delete", to: Routes.authorized_application_path(@conn, :delete, application), method: :delete, data: [confirm: "Are you sure?"], class: "btn btn-danger btn-xs" %></span>
```

Comment:
Gettext both delete and "Are you sure?"

Reply:
The Ok and Cancel button are translated by the browser so no worries about those.